### PR TITLE
override config from env

### LINKED ISSUE #101: Postgres secrets should be set separately
Currently, postgres secrets are set in `config.toml` with plane text:

```
[storage]
driver = "postgres"
datasource = "postgres://user:password@db_host/db_name?sslmode=disable"
```

Section `storage` should be set either via environment variables or via another config file in order to keep secrets separately.

### PR DESCRIPTION
**Pull request by Sergey1011010**
# Description

override config from env

Fixes #101 

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

## Testing steps
Please describe how the change was tested locally. If, for some reason, the testing was not done or not done fully, please describe what are the testing steps.

`make test` & `make integration_tests`

### CODE REVIEW

File: aggregator_test.go [hunk 1]
```diff
@@ -94,3 +95,30 @@ func TestStartStorageConnection(t *testing.T) {
...
 }
+
+func TestLoadConfigurationOverrideFromEnv(t *testing.T) {
+	err := os.Unsetenv("INSIGHTS_RESULTS_AGGREGATOR_CONFIG_FILE")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	main.LoadConfiguration("tests/config1")
+
```
Comment (tisnik) [inline aggregator_test.go]: I'd add Os.Unsetenv() just to be sure that other tests won't affect this one

File: README.md [hunk 2]
```diff
@@ -36,6 +36,33 @@ All packages developed in this project have documentation available on [GoDoc se
...
+driver = "sqlite3"
+datasource = "./aggregator.db"
+...
+```
+
+and environment variables 
+
+```
+INSIGHTS_RESULTS_AGGREGATOR__STORAGE__DRIVER="postgres"
+INSIGHTS_RESULTS_AGGREGATOR__STORAGE__DATASOURCE="postgresql://user:password@localhost:5432/dbname?param1=value1"
```
Comment (quarckster) [inline README.md]: I would prefer several variables and then build the URL from them:
```
INSIGHTS_RESULTS_AGGREGATOR__DB_USERNAME
INSIGHTS_RESULTS_AGGREGATOR__DB_PASSWORD
INSIGHTS_RESULTS_AGGREGATOR__DB_HOST
```

File: aggregator_test.go (hunk 1 above)
Comment (Sergey1011010) [reply aggregator_test.go]: actually, yes, it's better to clean the env completely before the test. Thanks

**Comment by bond95** [comment]
@Sergey1011010 please, rebase

**Comment by Sergey1011010** [comment]
@bond95 done

**Pull request merged**